SwiftCompilerSources/Sources/Optimizer/PassManager/PassUtils.swift

```diff
@@ -84,6 +89,14 @@ struct PassContext {
 
     RefCountingInst_setIsAtomic(instruction.bridged, isAtomic)
   }
+
+  func getDealloc(for type: Type) -> Function? {
```

Comment:
Please rename this function to `getDestructor(ofClass: Type)`.
We use the term "Destructor" also in the AST, e.g. `DestructorDecl`.
And as we don't have separate swift types for the different AST types (we just pack everything into `Type`), it should be clear from the label that the argument should be a class type.